src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/features.yml

```diff
@@ -1,19 +1,27 @@
-admin_features_create:
-  path: /new
-  methods: [ GET, POST ]
-  defaults:
-    _controller: PrestaShopBundle\Controller\Admin\Sell\Catalog\FeatureController::createAction
-    _legacy_controller: AdminFeatures
-    # _legacy_link: AdminFeatures:addfeature
+#admin_features_index:
```

Comment:
```suggestion
# admin_features_index:
```

Reply:
Can you please add a space in all comments It's not blocking but it would avoid us having lint warning in all the future PRs
Thanks

Reply:
Besides you don't need to comment the whole route (same for previous routes for create edit) You just need to comment the `_legacy_link` attribute This way the page is accessible via its URL but it's not used in the BO for links or redirection

This is how we keep the partially migrated page so far, so I think we can use the same principle for this page as well